### PR TITLE
Photo uploading

### PR DESCRIPTION
**Pull request by NadiaKishchuk**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by Kasterov — APPROVED** [review]
Well done!

File: src/features/AdminPage/NewStreetcode/MainNewStreetcode.component.tsx [hunk 1]
```diff
@@ -27,23 +29,25 @@ const NewStreetcode = () => {
...
                         <MainBlockAdmin form={form} />
                         <TextBlock />
                     </Form>
                     <InterestingFactsBlock />
-                    <ArtGalleryBlock />
-                    <RelatedFiguresBlock />
-                    <TimelineBlockAdmin />  
-                    <ForFansBlock />
+                    <ArtGalleryBlock arts={arts} setArts={setArts} />
+                    {/*  <RelatedFiguresBlock /> */}
```
Comment (Tysyatsky) [inline src/features/AdminPage/NewStreetcode/MainNewStreetcode.component.tsx:45]: remove comment 🥺

**Review by Tysyatsky — APPROVED** [review]
💯 result.

**Review by MaksBrat — APPROVED** [review]
Great job!

**Pull request merged**